tools/lkl/Makefile

```diff
@@ -4,6 +4,12 @@
 .SUFFIXES:
 MAKEFLAGS += -r --no-print-directory
 
+ifneq ($(LKL_FUZZING),)
```

Comment:
Can we move this to Makefile.autoconf?

Reply:
Sure, I moved this section which conditionally defines `kasan, LLVM, CROSS_COMPILE and KCONFIG` to Makefile.autoconf. The only thing which I'm not sure is the definition of `KCONFIG` which is now defined both in the Makefile (`KCONFIG?=defconfig`) and in Makefile.autoconf. What do you think about it. Should I also move this definition `KCONFIG?=defconfig` in Makefile.autoconf or leave it like that?

Reply:
Thank you. I think `KCONFIG?=defconfig` is fine in Makefile as we can override it from Makefile.autoconf if needed, exactly as you did for the fuzzers config.